bigframes/operations/__init__.py

```diff
@@ -204,6 +204,7 @@ def create_ternary_op(
 month_op = create_unary_op(name="month", type_rule=op_typing.INTEGER)
 quarter_op = create_unary_op(name="quarter", type_rule=op_typing.INTEGER)
 second_op = create_unary_op(name="second", type_rule=op_typing.INTEGER)
+strftime_op = create_unary_op(name="strftime", type_rule=op_typing.STRING)
```

Comment:
Sorry, this is a bit confusing, but `create_unary_op` creates a class. So this and the class definition below are redundant. Since we need the `date_format` parameter, we need the full class declaration, so lets delete this line.

Reply:
Oh, that makes sense! Thanks for explaining this :-)